src/security-council-mgmt/SecurityCouncilUpgradeAction.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "./interfaces/IGnosisSafe.sol";
+import "./SecurityCouncilMgmtUtils.sol";
+
+/// @notice Action contract for updating security council members. Used by the security council management system.
+contract SecurityCouncilUpgradeAction {
+    address internal constant SENTINEL_OWNERS = address(0x1);
+
+    IGnosisSafe public immutable securityCouncil;
+
+    constructor(address _securityCouncil) {
+        securityCouncil = IGnosisSafe(_securityCouncil);
+    }
+
+    /// @notice updates members of security council multisig to match provided array
+    function updateMembers(address[] memory _updatedMembers) external {
+        // always preserve current threshold
+        uint256 threshold = securityCouncil.getThreshold();
+
+        address[] memory membersToAdd = new address[](6);
+        uint8 membersToAddCount = 0;
```

Comment:
We could do without these uints and the new arrays by adding and removing inline. If we do that it's probably best to remove first then so that we doubly iterate the list. Eg:
```
      for (uint256 i = 0; i < _updatedMembers.length; i++) {
            if (!securityCouncil.isOwner(_updatedMembers)) {
                _addMember(_updatedMembers[i], threshold)
            }
        }
```

Reply:
hrm, can't remove first b/c you can't remove to below the threshold

Reply:
..but yeah i think we can just add/remove inline? i swear there was a reason i did it this way but can't justify it now, so